src/index.html

```diff
@@ -5,16 +5,118 @@
+
+        <p class="user-profile-page__header__description">
+          This is a design that you will need to code up and impress us.
+        </p>
+
+        <a href="#contact-form__footer" class="user-profile-page__header__arrow-button"></a>
+      </header>
+
+      <div class="user-profile-page__form-and-footer-container">
+        <form class="user-profile-page__form">
+          <div class="input-group">
+            <label for="name"> Name </label>
+            <div class="input-group__content">
+              <input id="name" type="text" name="name" placeholder="Kendall Jenner" />
+            </div>
+          </div>
+
+          <div class="input-group">
+            <label> Gender </label>
+            <div class="input-group__content">
+              <label class="radio-input">
+                <input type="radio" name="gender" value="male" />
+                <span>Male</span>
+              </label>
+              <label class="radio-input">
+                <input type="radio" name="gender" value="female" />
+                <span>Female</span>
+              </label>
+            </div>
+          </div>
+          <div class="input-group">
+            <label for="birthday">Date of Birth</label>
+            <input type="date" id="birthday" name="birthday" value="1983-02-10" />
+          </div>
+
+          <div class="input-group">
+            <label for="email"> Email </label>
+            <input id="email" type="email" name="email" placeholder=" Enter your email" required />
+          </div>
+
+          <div class="input-group">
+            <label class="input-group__label" for="quantity">Mobile</label>
+            <input type="number" id="quantity" name="quantity" min="1" max="11" placeholder="+91-98765-43210" />
+          </div>
+
+          <div class="input-group">
+            <label for="customer-id"> Customer ID </label>
+            <input id="customer-id" type="text" name="customer-id" placeholder="576802-ERD0348 45" required />
+          </div>
+
+          <div class="input-group__membership">
+            <label> Membership </label>
+            <div class="input-group__content">
+              <label class="radio-input">
+                <input type="radio" name="membership" value="classic" />
+                <span>Classic</span>
+              </label>
+              <label class="radio-input">
+                <input type="radio" name="membership" value="silver" />
+                <span>Silver</span>
+              </label>
+              <label class="radio-input">
+                <input type="radio" name="membership" value="gold" />
+                <span>Gold</span>
+              </label>
+            </div>
+          </div>
+
+          <div class="user-profile-page__form__buttons">
+            <button type="reset" class="user-profile-page__form__buttons-cancel">
+              Cancel
+            </button>
+
+            <button type="submit" class="user-profile-page__form__buttons-save">
```

Comment:
@3aPretorius The class name `user-profile-page__form__buttons-save` should probably be renamed to `user-profile-page__form__button--save` instead. Or maybe even just `.button--save` The sass names would then also need to be updated.